sycl/test-e2e/GroupAlgorithm/SYCL2020/sort.cpp

```diff
@@ -243,6 +243,240 @@ void RunJointSort(sycl::queue &Q, const std::vector<T> &DataToSort,
+               oneapi_exp::sort_key_value_over_group(
+                   oneapi_exp::group_with_scratchpad(
+                       Group,
+                       sycl::span{ScratchPtrDefault, LocalMemorySizeDefault}),
+                   sycl::span<U, /*ElementsPerWorkItem*/ 1>{
+                       &AccKeysToSort0[GlobalLinearID],
+                       &AccKeysToSort0[GlobalLinearID] + 1},
+                   sycl::span<T, /*ElementsPerWorkItem*/ 1>{
+                       &AccDataToSort0[GlobalLinearID],
+                       &AccDataToSort0[GlobalLinearID] + 1}); // (4)
+
+             oneapi_exp::sort_key_value_over_group(
+                 oneapi_exp::group_with_scratchpad(
+                     Group,
+                     sycl::span{ScratchPtrDefault, LocalMemorySizeDefault}),
+                 sycl::span<U, /*ElementsPerWorkItem*/ 1>{
+                     &AccKeysToSort1[GlobalLinearID],
+                     &AccKeysToSort1[GlobalLinearID] + 1},
+                 sycl::span<T, /*ElementsPerWorkItem*/ 1>{
+                     &AccDataToSort1[GlobalLinearID],
+                     &AccDataToSort1[GlobalLinearID] + 1},
+                 Comp, {}); // (5)
+
+             oneapi_exp::sort_key_value_over_group(
+                 Group,
+                 sycl::span<U, /*ElementsPerWorkItem*/ 1>{
+                     &AccKeysToSort2[GlobalLinearID],
+                     &AccKeysToSort2[GlobalLinearID] + 1},
+                 sycl::span<T, /*ElementsPerWorkItem*/ 1>{
+                     &AccDataToSort2[GlobalLinearID],
+                     &AccDataToSort2[GlobalLinearID] + 1},
+                 oneapi_exp::default_sorters::group_key_value_sorter<
+                     T, U, Compare, /*ElementsPerWorkItem*/ 1>(sycl::span{
+                     ScratchPtrDefault, LocalMemorySizeDefault})); // (6)
+
+             // Each sub-group should use it's own part of the scratch pad
+             const size_t ScratchShiftRadix =
+                 UseGroup == UseGroupT::SubGroup
+                     ? id.get_sub_group().get_group_linear_id() *
+                           LocalMemorySizeRadix
+                     : 0;
+             std::byte *ScratchPtrRadix = &ScratchRadix[0] + ScratchShiftRadix;
+
+             /* // Radix doesn't support custom types */
+             if constexpr (!std::is_same_v<CustomType, T>)
+               oneapi_exp::sort_key_value_over_group(
+                   Group,
+                   sycl::span<U, /*ElementsPerWorkItem*/ 1>{
+                       &AccKeysToSort3[GlobalLinearID],
+                       &AccKeysToSort3[GlobalLinearID] + 1},
+                   sycl::span<T, /*ElementsPerWorkItem*/ 1>{
+                       &AccDataToSort3[GlobalLinearID],
+                       &AccDataToSort3[GlobalLinearID] + 1},
+                   RadixSorterT(sycl::span{ScratchPtrRadix,
+                                           LocalMemorySizeRadix})); // (6) radix
+           });
+     }).wait_and_throw();
```

Comment:
Here we don't test the `striped` property. Could you please add one API call with the `striped` property?